Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -7136,6 +7143,9 @@ on revIDESaveStack pStackID
    
    lock messages
    
+   local tStackFilename
+   put the effective filename of stack tStackName into tStackFilename
+   
    local tStackFileVersion
    
    if the cPreserveStackVersion of stack "revPreferences" is true and the cREVGeneral["stackfileversion"] of stack tStackName is not empty then
```

Comment:
No, I think you're right actually - we can get rid of all occurrences of `cRevGeneral["stackfileversion"]`